### ISSUE TITLE
Javadoc-Fails to generate Javadoc

### ISSUE THREAD

**Issue by KarlHanEdn** [CMPUT301W22T35/QRcode-Quest#70]
I looked into javadoc generation and it seems after commit https://github.com/CMPUT301W22T35/QRcode-Quest/commit/7cb9f9e9f9564e1c60ecdf02f67e10e89bed5deb which introduces databindings "gradle javadoc" fails with package [package].databinding does not exist.

My Android Studio version is Arctic Fox. Here are the minimum steps to replicate the error:
1. create empty activity project
2. copy javadoc patch on eclass to the end of build.gradle(:app)
3. modify build.gradle(:app) to add inside android {}:
  BuildFeatures {
    viewBinding true
  }
4. Add an ActivityMainBinding object and auto import the binding class
5. double ctrl to run "gradle javadoc" gives the error above

**Comment by KarlHanEdn** [comment]
After searching online for quite a while I can't seem to find anything that points to possible cause of the error. Javadoc generation is not very urgent so I will leave it for now.

**Comment by RottenFishbone** [comment]
I've been searching for the past bit. As far as I can tell so far, its because it is not including the android.jar in the command its calling. 

Not sure why, or if that is true yet. Working on running with that atm.

**Comment by RottenFishbone** [comment]
Following [this](https://stackoverflow.com/questions/70953332/android-failed-to-generate-javadoc-with-android-studio-bumblebee) 'solved' the problem for me.

It's an awful workaround but given we only need to generate once at the end, I think it'll be fine.

**Comment by RottenFishbone** [comment]
I've been looking for about an hour now and I also can't find any results. Binding isn't required, its just much cleaner. Worst case we just swap back to the older way of doing it.

**Comment by KarlHanEdn** [comment]
Yeah I've spent a lot of time searching for it, probably it's not worthwhile. The databinding is not part of our application's interfaces so if we only need to generate javadoc once at the end, we can swap it out-generate-and go back.

**Comment by KarlHanEdn** [comment]
TA was able to replicate the issue as well, and suggested we should probably use the workaround.
